env/netbox.env

```diff
@@ -15,6 +15,7 @@ NAPALM_TIMEOUT=10
 MAX_PAGE_SIZE=1000
 REDIS_HOST=redis
 REDIS_PASSWORD=H733Kdjndks81
+REDIS_SSL=false
```

Comment:
I think this is not necessary, as the default is `false` anyway, and we don't have all the variables in the list either.

Reply:
I was on the fence; initially I almost made the pull request with suggested change of making it true by default in the env file since using a password without SSL is still pretty fundamentally insecure (the AUTH command sends it as plaintext). Redis ElastiCache in AWS doesn't even permit using auth without SSL. But I wasn't sure how widely common it would be for people to actually have their Redis servers configured for SSL. Thus explicitly calling out the unsafe default combination of parameters here seemed like a compromise.

Reply:
🤔 Fair point.

Turning TLS on by default would not work, because the Redis which is started with the default `docker-compose.yml` configuration of netbox-docker does not support TLS.

Reply:
I dislike that it's called `REDIS_SSL`, as SSL is dead and was superseded by TLS almost 20 years ago. What do you think about changing the name to `REDIS_TLS`?

Reply:
I'm fine with either name, it's just that for better or worse the redis-py library kwarg for this parameter is named 'ssl'. I just tried to thread it back up the stack with the matching name. So django-rq (the real root dependency here) and the main netbox project have already accepted the pull requests with the REDIS_SSL naming. Thus the risk (albeit small) to using a different name here would be the potential for confusion, especially in terms of making the mapping between netbox-docker and netbox itself.

Reply:
¯\_(ツ)_/¯